Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/StubGenerator.kt

```diff
@@ -732,32 +732,31 @@ class StubGenerator(
     }
 
     private fun integerLiteral(size: Int, isSigned: Boolean, value: Long): String? {
-        if (!isSigned) {
-            val signedLiteral = integerLiteral(size, true, value)
-            val converter = when (size) {
-                1 -> "toUByte"
-                2 -> "toUShort"
-                4 -> "toUInt"
-                8 -> "toULong"
-                else -> return null
+        return if (isSigned) {
+            if (value == Long.MIN_VALUE) {
+                return "${value + 1} - 1" // Workaround for "The value is out of range" compile error.
```

Comment:
:)

Reply:
And Kotlin still don't provide a way to define `Long.MIN_VALUE` with a single literal, so this workaround is still required!